taui/src/components/neighborhood-details.js

```diff
@@ -241,7 +241,18 @@ export default class NeighborhoodDetails extends PureComponent<Props> {
               type={isFavorite ? 'star' : 'star-o'}
               onClick={(e) => setFavorite(id, userProfile, changeUserProfile)}
             />
-            <div className='neighborhood-details__name'>{town} &ndash; {id}</div>
+            <div className='neighborhood-details__name'>
+              {town} &ndash; {id}
+              {ecc &&
+                <div
+                  className='neighborhood-details__ecc'
+                  data-tip={message('Tooltips.ExpandedChoice')}
```

Comment:
These tooltips are still showing off the right end of the card, although the label is now left-aligned. Was that intentional?

Reply:
Fixed!